pynxtools_xps/config/config_vms.json

```diff
@@ -45,11 +45,20 @@
          "identifier":"@eln",
          "model":"@eln"
        },
-       "associated_beam":"@link:/entry/instrument/beam_probe",
-       "source_analyzer_angle":"source_analyzer_angle",
-       "source_analyzer_angle/@units":"degree"
+       "associated_beam":"/entry/instrument/beam_xray",
+       "depends_on": "/entry/instrument/source_xray/transformations/source_analyzer_angle",
+       "TRANSFORMATIONS[transformations]": {
+         "AXISNAME[source_analyzer_angle]":"@xps_token:source/source_analyzer_angle",
+         "AXISNAME[source_analyzer_angle]/@transformation_type": "rotation",
+         "AXISNAME[source_analyzer_angle]/@units":"degree",
+         "AXISNAME[source_analyzer_angle]/@vector": [
+           0,
+           0,
+           1
```

Comment:
This should ideally also contain a `depends_on` (can be `.` if this refers to the base coordinate system)

Reply:
Makes sense, I added it in all config files that contain transformations.